docs/01_developers/build_image.md

```diff
@@ -0,0 +1,40 @@
+# Building a Garden Linux Image
+
+Building a Garden Linux image is a straightforward process, thanks to the new Garden Linux builder. This guide will walk you through the steps to create your custom Garden Linux image.
```

Comment:
```suggestion
Building a Garden Linux image is a straightforward process, thanks to the Garden Linux builder. This guide will walk you through the steps to create your custom Garden Linux image.
```

I'd be careful with using words like 'new' in timeless documents.. the builder is new now, but is this relevant for the reader?